src/components/UI/Loader/Loader.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import { Loader as MantineLoader, useMantineTheme } from "@mantine/core";
+import { loaderVariants } from "./styles";
+
+type LoaderSize = "small" | "medium" | "large" | "extraLarge";
+
+interface LoaderProps {
+  size?: LoaderSize;
+  variant?: keyof typeof loaderVariants;
+  className?: string;
+}
+const sizeMapper: Record<LoaderSize, "xs" | "sm" | "md" | "lg" | "xl"> = {
+  small: "sm",
+  medium: "md",
+  large: "lg",
+  extraLarge: "xl",
+};
+
+export const Loader: React.FC<LoaderProps> = ({
```

Comment:
Can you use `FC`  and not the static function from react in here ? meaning import fc from react and use it